`codemirror-extension` example fails to build

## Description

Running `jlpm run build` locally in the `codemirror-extension` example locally results in an error with the following message:
```bash
src/index.ts:46:25 - error TS18046: 'step' is of type 'unknown'.

46       if (line.number % step === 0) {
                           ~~~~


Found 1 error in src/index.ts:46
```

## Context

JL 4 typical extension conda/mamba/micromamba setup, i.e., same setup as `binder/environment.yml` in the extension template.

--- a/codemirror-extension/src/index.ts
+++ b/codemirror-extension/src/index.ts
@@ -38,7 +38,7 @@ const stripe = Decoration.line({
 
 // Create the range of lines requiring decorations
 function stripeDeco(view: EditorView) {
-  const step = view.state.facet(stepSize);
+  const step = view.state.facet(stepSize) as number;
   const builder = new RangeSetBuilder<Decoration>();
   for (const { from, to } of view.visibleRanges) {
     for (let pos = from; pos <= to; ) {

